modules/ROOT/pages/verify-nfs.adoc

```diff
@@ -0,0 +1,187 @@
+= Verify NFS Server Requirements
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Private Cloud Edition supports the Network File System (NFSv4) protocol.
+
+== Verify the Required Ports are Open
+
+The NFS server requires ports 2049 and 111 to be open.
+ 
+Use the netcat tool to verify the required ports are open. The netcat tool is a computer networking utility used to read and write to network connections using TCP or UDP. 
+
+To verify the required ports are open, execute the following commands:
+
+- `nc -zvu <host_name> 111`
+- `nc -zv <host_name> 2049`
+
+When the responses are successful you receive a response similar to the following:
+
+----
+[ec2-user@ip-10-1-1-97 ~]$ nc -zv <nfs_host_name> 2049
+Ncat: Version 7.50 ( https://nmap.org/ncat )
+Ncat: Connected to 10.1.0.155:2049.
+Ncat: 0 bytes sent, 0 bytes received in 0.01 seconds.
+----
+
+== Mount the NFS Server
+
+Mount the NFS server in a temporary directory on the host. 
+
+. Switch to root by typing: +
+`sudo su`
+. To mount the NFS in a temporary directory, run the following commands and replace both the `<nfs-server>` and the `<nfs-path>` with your own values. +
+Run: +
+`mkdir -p /mnt/home` +
+and then: +
+`mount -t nfs4 -o proto=tcp,port=2049 <nfs-server>:<path> /mnt/home/` +
+If the mount fails, you do not have the required permissions. 
+
+== Validate NFS Performance
+
+Perform careful analysis of your environment, both from the client and server point of view for optimal NFS performance.
+
+These are the high level steps for validating NFS performance:
+
+. Perform some basic operations to test the performance and latency of your NFS server. Specifically, these operations will: +
+* Transfer data from the client (Anypoint Platform environment) to the NFS server 
```

Comment:
These bullet items need a period because they are sentences (when combined with the lead in)?